<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>80000.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="input_peso">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>149</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 98%;
padding: 10px;
border-color:white;
border:2px solid white;
background-color: blue;
color: white;
font-family: 'Orbitron', sans-serif;</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_resultado">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>149</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 98%;
padding: 10px;
border-color:white;
border:2px solid white;
background-color: blue;
color: white;
font-family: 'Orbitron', sans-serif;</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calcular">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
width: 98%;
padding: 10px;
border-color:white;
border-top-left-radius: 15px;
border-bottom-right-radius: 12px;
border:2px solid navy;;
background-color: black;
color: white;
font-family: 'Orbitron', sans-serif;
}
QPushButton:hover{
	background-color:navy;
    color:white;
}
</string>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_muito_abaixo_do_peso">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(198, 70, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Muito Abaixo do Peso&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_abaixo_do_peso">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>145</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 240, 107);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Abaixo do Peso&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_normal">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>181</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(153, 193, 241);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Peso Normal&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_acima_do_peso">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>216</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 138, 221);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Acima do Peso&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_obesidade_i">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>251</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 97, 81);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Obesidade I&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_obesidade_ii">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>286</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 51, 59);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Obesidade II&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_obesidade_iii">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>320</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Obesidade III&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_altura">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>149</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 98%;
padding: 10px;
border-color:white;
border:2px solid white;
background-color: blue;
color: white;
font-family: 'Orbitron', sans-serif;</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_limpar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>110</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
width: 98%;
padding: 10px;
border-color:white;
border-top-left-radius: 15px;
border-bottom-right-radius: 12px;
border:2px solid navy;;
background-color: black;
color: white;
font-family: 'Orbitron', sans-serif;
}
QPushButton:hover{
	background-color:navy;
    color:white;
}
</string>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>561</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
width: 98%;
padding: 10px;
border-color:white;
border-top-left-radius: 15px;
border-bottom-right-radius: 12px;
border:2px solid cyan;
background-color: #323232;
color: white;
font-family: 'Orbitron', sans-serif;
letter-spacing:4px;
}
QPushButton:hover{
	background-color:#323232;
    color:black;
}
</string>
    </property>
    <property name="text">
     <string>CALCULADORA DE IMC (INDICE DE MASSA CORPÃ“REA)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>109</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: cyan;
letter-spacing:4px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;PESO EM KG&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>109</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: cyan;
letter-spacing:4px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ALTURA MTS&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: cyan;
letter-spacing:4px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;RESULTADO&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="seta">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>79</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>seta.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_faixa">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 98%;
padding: 10px;
border-color:white;
border-top-left-radius: 15px;
border-bottom-right-radius: 12px;
border:2px solid blue;
background-color: #323232;
color: white;
font-family: 'Orbitron', sans-serif;
letter-spacing:4px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>input_peso</zorder>
   <zorder>label_resultado</zorder>
   <zorder>btn_calcular</zorder>
   <zorder>label_muito_abaixo_do_peso</zorder>
   <zorder>label_abaixo_do_peso</zorder>
   <zorder>label_normal</zorder>
   <zorder>label_acima_do_peso</zorder>
   <zorder>label_obesidade_i</zorder>
   <zorder>label_obesidade_ii</zorder>
   <zorder>label_obesidade_iii</zorder>
   <zorder>input_altura</zorder>
   <zorder>pushButton_limpar</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>seta</zorder>
   <zorder>label_faixa</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
